frontend/toolkit-webapp/src/pages/Dashboard.js

```diff
@@ -117,7 +121,13 @@ const Dashboard = (props) => {
 
                     </Grid>
                 </div>
-                {/* <Pagination count={6} page={page} onChange={handlePage} variant="outlined" shape="rounded" /> */}
+                
+                { totalCourses !== courses.length && (courses.length - totalCourses > cardAmount) && 
```

Comment:
not working for last load since courses.length - totalCourses will be less than cardAmount